code/particle/effects/GenericShapeEffect.h

```diff
@@ -179,6 +185,10 @@ class GenericShapeEffect : public ParticleEffect {
 			saw_deprecated_effect_location = true;
 		}
 
+		if (optional_string("+Velocity Inherit:")) {
```

Comment:
To me, this name seems a bit weird. It seems to indicate that it is a simple boolean value but instead is a factor on the inherited velocity.
Maybe something like `+Velocity Factor:` would be more appropriate? Also, you could make use of a random range here to allow for some variability if desired by the modder.

Reply:
`Parent velocity factor:` maybe?